sdv/utils.py

```diff
@@ -164,3 +164,8 @@ def load_data_from_csv(filepath, pandas_kwargs=None):
     pandas_kwargs = pandas_kwargs or {}
     data = pd.read_csv(filepath, **pandas_kwargs)
     return data
+
+
+def groupby_list(list_to_check):
+    """Return the list element if the length is 1 and the list otherwise."""
```

Comment:
I would use something more descriptive as -> Return the first element of the list if the length is 1 else the entire list.

Reply:
Yes good idea, done in [41690bd](https://github.com/sdv-dev/SDV/pull/1374/commits/41690bd97a50e0ba17921f6293d4385da17d96a8).